files/ie10-viewport/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Twitter"
```

Comment:
I think its best to set the author as "Alexandr Kondrashov" because he controls the repo and there is no guarantee he wont modify the files later. I dont want to confuse people